test/feed.test.js

```diff
@@ -218,6 +224,49 @@ describe('Post data class tests', () => {
       expect(clearedFeed2.lastModified).toBe(null);
       expect(clearedFeed3.etag).toBe(null);
       expect(clearedFeed3.lastModified).toBe(null);
+      // Teardown removing the added feed
+      await Promise.all([await feed.delete(), await feed2.delete(), await feed3.delete()]);
+    });
+
+    test('Flagged feed should appear in t:feeds:flagged and not t:feeds', async () => {
+      let feed = new Feed(data.author, data.url, data.user, data.link);
+      let feed2 = new Feed(data2.author, data2.url, data2.user, data2.link);
+      let feed3 = new Feed(data3.author, data3.url, data3.user, data3.link);
+
+      // Check all three feeds are created
+      await Promise.all([Feed.create(feed), Feed.create(feed2), Feed.create(feed3)]);
+      feed = await Feed.byId(await Feed.create(feed));
+      feed2 = await Feed.byId(await Feed.create(feed2));
+      feed3 = await Feed.byId(await Feed.create(feed3));
+      let unFlaggedFeeds = await Feed.all();
+      expect(unFlaggedFeeds.length).toBe(3);
+
+      // test setFlagged()
+      await Promise.all([feed.setFlagged(), feed2.setFlagged()]);
+      unFlaggedFeeds = await Feed.all();
+      expect(unFlaggedFeeds.length).toBe(1);
+
+      let flaggedFeeds = await Feed.allFlagged();
+      expect(flaggedFeeds.length).toBe(2);
+
+      // Flagged feeds should have same data as feed + feed2
+      expect(flaggedFeeds[0].id).toBe(feed.id);
```

Comment:
Can you do this check without relying on the order?  I'm not sure we can always assume we'll get back the values in these positions.  A set has no order.

Reply:
I changed it to the below
```
expect(flaggedFeeds.some((flaggedFeed) => flaggedFeed.id === feed.id)).toBe(true);
expect(flaggedFeeds.some((flaggedFeed) => flaggedFeed.id === feed2.id)).toBe(true);
```